docker/compute_worker/compute_worker.py

```diff
@@ -371,6 +374,21 @@ def _get_bundle(self, url, destination):
 
             # Input from submission (or submission + ingestion combo)
             docker_cmd += ['-v', f'{self.input_dir}:/app/input']
+            if self.detailed_results_url and self.is_scoring and kind == 'program':
+                # we have a detailed results url, we are in the scoring step, and we aren't the ingestion program
+                detail_path = os.path.join(self.output_dir, "detailed_results.html")
+
+                # Is there a better way to create the entire path without having to mkdir each incremental step?
```

Comment:
If I'm understanding, this may help?

https://docs.python.org/3/library/os.html#os.makedirs

> Recursive directory creation function. Like mkdir(), but makes all intermediate-level directories needed to contain the leaf directory.

...neat!

Reply:
not quite what I need. since I just need to make a file one folder in. I would still need to `mkdirs()` and `open().close()`